src/doc/HOWTO/EMUfailure.sgml

```diff
@@ -358,7 +358,63 @@ the startup screen it runs fine though.
 </listitem>
 </itemizedlist>
 </sect2>
+</sect1>
 
+<sect1>
+<title>Differences in behaviour between Dosemu and Dosemu2</title>
+<para>
+The following differences may be apparent if you have previously been
+using Dosemu.
+</para>
+<sect2>
+<title>
+Filesystems
+</title>
+
+<sect3>
+<title>
+MFS
+</title>
+<para>
+Network device DOS filesystem that provides read/write access to the host filesystem.
+</para>
+<itemizedlist>
+<listitem>
+<para>
+Dosemu2 does not redirect a drive to arbitrary host paths on the command
+line of the emufs.sys driver when it is loaded in config.sys. Use the lredir2
+command to accomplish this in autoexec.bat instead.
+</para>
+</listitem>
+<listitem>
+<para>
+Dosemu2 does not support SUBST / JOIN when using its long filename support
+</para>
+</listitem>
+</itemizedlist>
+</sect3>
+<sect3>
+<title>
+FATFS
+</title>
+<para>
+Block device DOS filesystem that provides read only access to the host filesystem.
+</para>
+<itemizedlist>
+<listitem>
+<para>
+PC-DOS 3.x and MS-DOS 3.x are automatically limited to maximum of 2 drives or
+DOS would panic at boot.
```

Comment:
This is more of a bug-fix, not a regression.

Reply:
Okay I guess I just remove it then since the file is called 'emufailures'?

Reply:
Yes, if we mention all differences between
dosemus 1 and 2, we can as well write the book. :)
We only need to note an obvious regressions,
which are fortunately quite few.